helpdesk/email.py

```diff
@@ -581,18 +605,33 @@ def object_from_message(message, queue, logger):
         if beautiful_body:
             try:
                 body = beautiful_body.text
+                full_body = body
             except AttributeError:
                 pass
         if not body:
             body = ""
 
+    if getattr(django_settings, 'HELPDESK_ALWAYS_SAVE_INCOMING_EMAIL_MESSAGE', False):
```

Comment:
Currently we already save a `email_html_body.html`. If that file isn't enough sometimes, then it would be better not to save `email_html_body.html` at all and save the `.eml` file by default. The only point of looking at `email_html_body.html` is that you would need to see missing formatting. In which case the origional `.eml` file should be even better. Indeed, I don't see the need for this to be a setting, it should be on by default.

Reply:
my initial thinking about all that settings was to keep backwards compatibility and don't surprise existing users who don't need that; if it's not a problem then yes, I'd replace `email_html_body.html` by these `.eml` files because they are bulletproof source of truth and make this behaviour default.